asciidoc-confluence-publisher-client/src/main/java/org/sahli/asciidoc/confluence/publisher/client/http/ConfluenceRestClient.java

```diff
@@ -379,6 +380,7 @@ private static CloseableHttpClient defaultHttpClient(boolean disableSslVerificat
         RequestConfig requestConfig = RequestConfig.custom()
                 .setConnectionRequestTimeout(20 * 1000)
                 .setConnectTimeout(20 * 1000)
+                .setCookieSpec(CookieSpecs.STANDARD)
```

Comment:
Code style: please use static import for `STANDARD`

Reply:
done, thank you